AppCenter/AppCenter/Internals/Channel/MSOneCollectorChannelDelegate.m

```diff
@@ -9,7 +10,6 @@
 #import "MSOneCollectorIngestion.h"
 #import "MSSDKExtension.h"
 
-static NSString *const kMSOneCollectorGroupIdSuffix = @"/one";
 static NSString *const kMSOneCollectorBaseUrl =
```

Comment:
Can probably remove this TODO if we're not planning to address it.

Reply:
Not in scope for this PR but valid comment. Will do it separately.